server/src/main/java/org/elasticsearch/index/engine/LazySoftDeletesDirectoryReaderWrapper.java

```diff
@@ -274,4 +274,39 @@ private static class DelegatingCacheHelper extends org.apache.lucene.index.Filte
             super(delegate);
         }
     }
+
```

Comment:
This used to be on FieldExistsDocValuesQuery but has been removed in a refactor.